.travis.yml

```diff
@@ -25,7 +25,6 @@ matrix:
       language: objective-c
       osx_image: xcode8.2
       script:
-        - brew update
```

Comment:
This preceded the `brew install bats`, but [bats hasn't had a release since 2014](https://github.com/sstephenson/bats/releases), so in a pragmatic sense, I agree that it's unnecessary.